src/main/java/com/EstimatedDeliveryDate/Entity/Holiday.java

```diff
@@ -0,0 +1,41 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.io.Serializable;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/*Document object for the Holiday List*/
+
+@Setter
+@NoArgsConstructor
+@DynamoDBDocument
+public class Holiday  implements Serializable {
+	private static final long serialVersionUID = 1L;
+	private String holidayDate;
+	private Integer deliveryHoliday;
+	private Integer pickUpHoliday;
+	private Integer processingHoliday;
```

Comment:
Number here doesn't give a clear picture on what the object actually represents. In future, if someone goes through code it won't be clear on what the object represents.
private Boolean isPickUpHoliday gives better description of the object.